### PR TITLE
Update 16-motivation.md

### PR DESCRIPTION
**Pull request by raynamharris**
Address comments raised in Issue https://github.com/swcarpentry/instructor-training/issues/463

Please delete the text below before submitting your contribution. 

---

Thanks for contributing! If this contribution is for instructor training, please send an email to checkout@carpentries.org with a link to this contribution so we can record your progress. You’ve completed your contribution step for instructor checkout just by submitting this contribution.  

Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.  If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact Kate Hertweck (k8hertweck@gmail.com).  

---

### CODE REVIEW

File: _episodes/16-motivation.md [hunk 1]
```diff
@@ -135,15 +135,16 @@ to make sure they're doing at least a few of these things.
...
-> This is a great didactic tool, but it can be hard for learners to see how they can use it in their research.
-> After the grep lesson, I show a one liner that combines `head`, `grep`, `sort`, and `uniq`
->  to produce a ranked list of the most abundant sequences.
-> I emphasize that the students just learned each of the pieces
-> (see <https://wikis.utexas.edu/display/bioiteam/Scott's+list+of+linux+one-liners>).
-> This way, I connect my bioinformatics users with domain-specific examples
+>
+> Or, insert an example story about establishing value, which goes like this:
+>
+> In the [SWC Unix "Finding Things" episode](http://swcarpentry.github.io/shell-novice/07-find/), a haiku is used to to teach grep.
```
Comment (ChristinaLK) [inline _episodes/16-motivation.md]: Maybe put this story in quotes, as it's being told in the first person?

File: _episodes/16-motivation.md [hunk 2]
```diff
@@ -135,15 +135,16 @@ to make sure they're doing at least a few of these things.
...
+> This is a great didactic tool, but it can be hard for learners to see how it applies to research.
+> After the didatic example, I connect my bioinformatics users with domain-specific examples
 > using an authentic task that is relevant to their research.
+> I point users to [a list of one-line unix commands](https://wikis.utexas.edu/display/bioiteam/Scott's+list+of+linux+one-liners) consisiting of  `grep`, `sort`, `head`, `uniq`, and more to explore biological sequence data.
+> This emphasizes how they can apply what they just leared with haikus to real research probolems. 
+> (see <>).
```
Comment (ChristinaLK) [inline _episodes/16-motivation.md]: delete this line?

File: _episodes/16-motivation.md [hunk 3]
```diff
@@ -135,15 +135,16 @@ to make sure they're doing at least a few of these things.
...
-> This way, I connect my bioinformatics users with domain-specific examples
+>
+> Or, insert an example story about establishing value, which goes like this:
+>
+> In the [SWC Unix "Finding Things" episode](http://swcarpentry.github.io/shell-novice/07-find/), a haiku is used to to teach grep.
+> This is a great didactic tool, but it can be hard for learners to see how it applies to research.
+> After the didatic example, I connect my bioinformatics users with domain-specific examples
 > using an authentic task that is relevant to their research.
+> I point users to [a list of one-line unix commands](https://wikis.utexas.edu/display/bioiteam/Scott's+list+of+linux+one-liners) consisiting of  `grep`, `sort`, `head`, `uniq`, and more to explore biological sequence data.
+> This emphasizes how they can apply what they just leared with haikus to real research probolems.
```
Comment (ChristinaLK) [inline _episodes/16-motivation.md]: probolems -> problems

**Comment by raynamharris** [comment]
Thanks @ChristinaLK. I think I addresses all the points in your review.

**Comment by ErinBecker** [comment]
Looks good to me, thanks @raynamharris!

**Pull request merged**